cincinnati/src/plugins/internal/graph_builder/github_openshift_secondary_metadata_scraper/github_v3.rs

```diff
@@ -43,6 +43,14 @@ pub(crate) fn archive_entry_directory_name(org: &str, repo: &str, commit: &Commi
     format!("{}-{}-{}", &org, &repo, &commit.sha[0..7],)
 }
 
+/// Format a commit URL
+pub(crate) fn commit_url(org: &str, repo: &str, sha: &str) -> String {
+    format!(
+        "https://api.github.com/repos/{}/{}/commits/{}",
```

Comment:
This sort of thing will require restricted-network folks who want to use this plugin to use the GitHub API (vs. just hosting a Git repository over Git's native Git or HTTP(S) protocols).  I guess that's an existing requirement from #234, so we can always come back and revisit later, but just thought I'd mention it here.

Reply:
This plugin is the GitHub specific scraper. The idea behind decoupling the scraper and the parser is that the scraper can be replaced by something else, or could in fact just be skipped, where the files for the parser would be managed out-of-process somehow. All the possibilities here ;-)

Reply:
We should investigate if its worth supporting Github Enterprise - or a plain git plugin is more common. I think its fine to go as is atm

Reply:
> or a plain git plugin is more common

I think a plain git plugin would be most useful in the long-term, as it's easy to transfer to and set up in a network-restricted environment while preserving the metadata ( at _.git_). It will be easy to swap out this plugin out simply by configuration in the future, so we can leave it in place now and revisit later.

Reply:
We should revisit this from air-gap cluster point of view for sure.